braidpool-primitives/src/braid/mod.rs

```diff
@@ -26,14 +26,17 @@ type NumberOfBeadsUnorphaned = usize;
 #[derive(Clone, Debug, Serialize)]
 
 pub struct Braid {
-    beads: HashSet<BeadHash>,
-    tips: HashSet<BeadHash>,
-    cohorts: Vec<Cohort>,
+    pub(crate) beads: HashSet<BeadHash>,
```

Comment:
Add a timestamp field here. (When this bead arrived according to the node's clock) Sender timestamps are in the bead. We will use this to characterize our latency measurements.